rbd.c

```diff
@@ -134,6 +154,24 @@ static int tcmu_rbd_image_open(struct tcmu_device *dev)
 		goto set_cluster_null;
 	}
 
+#ifdef LIBRADOS_SUPPORTS_SERVICES
+	char daemon_buf[1024];
```

Comment:
This could be dynamically allocated as #200 has fix a similar issue to defence the stack clashing attack.